Utils/Dataflow/069_upload2es/load_data.sh

```diff
@@ -19,14 +30,26 @@ ES_CONFIG="${base_dir}/../../Elasticsearch/config/es"
 
 while [ -n "$1" ]; do
   case "$1" in
-    --config|-c)
+    -c|--config)
```

Comment:
This change must be done in a separate commit: it has nothing to do with custom EOB/EOM, it is just formatting.